### PR TITLE
FEAT: added liked videos functionality

### PR DESCRIPTION
**Pull request by zoyatanweer**
Files to check :
- api/videos.js
- pages/Liked/Liked.jsx
- context/ VideoContext.jsx/

### CODE REVIEW

File: src/components/VideoCard/VideoCard.jsx [hunk 1]
```diff
@@ -7,25 +10,48 @@ import {
...
+  const { videos } = videoState;
+
+  // const likeVideoToggleHandler = (video) => {
+  //   const index =
+  //     likedVideos.findIndex((item) => item._id === video._id) === -1;
+  // };
+
+  // }
+  // const isInLikedVideos =
+  //   likedVideos.findIndex((i) => i._id === videos._id) === -1 ? false : true;
```
Comment (srishtigautam21) [inline src/components/VideoCard/VideoCard.jsx:27]: Remove commented code

File: src/context/Auth/AuthContext.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    operation: authFetch,
+  } = useAxios();
+  useEffect(() => {
+    if (user === null) {
+      authFetch({
+        method: "post",
+        url: "/api/auth/login",
+        data: {
+          email: "adarshbalika@gmail.com",
+          password: "adarshbalika",
```
Comment (srishtigautam21) [inline src/context/Auth/AuthContext.jsx:20]: Change the login credentials from adarsh balika to something else

File: src/context/VideoContext.jsx [hunk 3]
```diff
@@ -121,5 +159,3 @@ const useVideo = () => useContext(VideoContext);
 //   };
```
Comment (srishtigautam21) [inline src/context/VideoContext.jsx:159]: remove commented code

File: src/pages/Liked/Liked.jsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+            })}
+          </div>
+        </div>
+      </div>
+    </>
+  );
+
+  //   const isInLikedVideos =
+  //     likedVideos.findIndex((i) => i._id === videos._id) === -1 ? false : true;
+  //   return videos.map((videos) => {
```
Comment (srishtigautam21) [inline src/pages/Liked/Liked.jsx:63]: Commented code

**Review by srishtigautam21 — APPROVED** [review]
The code looks good to merge with changes

**Review by Jia-A — APPROVED** [review]
Your Liked.css is empty I guess, you can delete it.
Everything is structured. Good to merge.

**Comment by zoyatanweer** [comment]
Thankyou for your suggestions :)

**Pull request merged**